swhkd/src/daemon.rs

```diff
@@ -490,14 +495,22 @@ pub fn set_command_line_args() -> Command<'static> {
         )
         .arg(arg!(-d - -debug).required(false).help("Enable debug mode."))
         .arg(
-            arg!(-D --device <DEVICE_NAME>)
+            arg!(-D --device <DEVICE_NAMES>)
                 .required(false)
                 .takes_value(true)
                 .multiple_occurrences(true)
                 .help(
-                    "Specific keyboard devices to use. Seperate multiple devices with semicolon.",
+                    "Specific keyboard devices to use. Separate multiple devices with semicolon.",
                 ),
-        );
+        )
+        .arg(arg!(-I --ignore-device <DEVICE_NAMES>)
```

Comment:
So I just tested this Pr, and it seems that the `--ignore-device` is seeming to cause some issue with `clap`. Using `--ignoredevice` actually fixes the problem.

<details>
<summary>Details</summary>
<pre>
thread 'main' panicked at swhkd/src/daemon.rs:506:14:
assertion `left == right` failed: Short flags should precede long flags
  left: Some("ignore")
 right: None
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
</pre>
</details>

This is the actual error if it helps.

I'm not sure if it's OS specific either, let me know if it's just me :)

Reply:
Hey, can you please tell me what OS and shell you're using?

Reply:
I'll go ahead and change this. @Shinyzenith thoughts on `--ignoredevice` vs `--ignore`?

Reply:
> Hey, can you please tell me what OS and shell you're using?

I'm currently using Ubuntu 23.10 with fish shell...

Although I have tested it on bash....very weird why it's happening

Reply:
--ignoredevice is fine!